.gitconfig

```diff
@@ -8,7 +8,10 @@
 	ah = commit --amend -C HEAD
 	b = branch
 	bd = branch -D
-	c = commit -m
+	c = commit
```

Comment:
This opens up the Vi editor which allows for multi-line comments, useful if you want to attach the JIRA ticket your commit is associated with to the git history